sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/subscriptions/SubscriptionsDbConnector.groovy

```diff
@@ -88,10 +88,14 @@ class SubscriptionsDbConnector implements SubscriptionDataSource {
     }
     
     private void addSubscription(Connection connection, int subscriberId, String projectCode) {
+
+          if (projectAlreadySubscribed(subscriberId, projectCode)) {
+              throw new DataSourceException("Subscriber is already subscribed to ${projectCode}")
```

Comment:
noooo this should not throw an exception ;)

Reply:
or just return from the method and do nothing
```suggestion
              return
```

Reply:
I adapted it according to your suggestion in our meeting :)